### ISSUE TITLE
Add peerDependencies

### ISSUE THREAD

**Issue by alfonsogarciacaro** [fable-elmish/react#1]
Latest dotnet-fable can read the peerDependencies when using the `add` command. So if we add the Fable dependencies as peerDependencies into fable-elmish-react package.json, users can just type `dotnet fable add fable-elmish-react@next` instead of `dotnet fable add fable-powerpack@next fable-react@next fable-elmish@next fable-elmish-react@next`.

I can PR this but I'm not sure how do you want to manage the versions of the peerDependencies, do you want to retrieve them from the other projects during the build process or write them somewhere? For other elmish projects I guess we can just use the same minor version (e.g. `^0.9`) but that may not work well for other dependencies like fable-react or fable-powerpack.

**Comment by et1975** [comment]
This is welcome news!
I'm not sure I understand the question though, wouldn't I just require certain peer dependency, in project.json?
- in fable-elmish-react: `peerDependencies: { "fable-elmish": "next" }`
- in fable-elmish: `peerDependencies: { "fable-powerpack": "next" }`
etc?

**Comment by alfonsogarciacaro** [comment]
Hmm, true, that would be another (and probably more sensible) way to do it. Unfortunately, the way [I implemented it for now](https://github.com/fable-compiler/Fable/blob/7287cd7d053dc5a08299cdd5290248ced5c02b33/src/dotnet/Fable.Tools/Main.fs#L192-L202) just asks the peerDependencies of the package name passed to `dotnet fable add`, and doesn't check the other packages recursively 😅  (A PR with something more elaborated would be welcome though.)

Basically, what I was trying to ask in my terrible English :wink: is whether you want to manually set the version of the peerDependencies (in your example, `"next"`) or do it automatically in the build script.

Actually, now that I check it, it seems the npm package version is not being taken from RELEASE_NOTES.md as I had imagined. So I guess setting the version of the peerDependencies manually should be ok for now.

**Comment by et1975** [comment]
To my English-as-a-2nd-language eyes it looked perfectly legit ;) my confusion was to do with the implication, which you cleared up, that you are not looking into package.json for peer dependencies.
Btw, i'm working on versioning from RELEASE_NOTES.md...

**Comment by et1975** [comment]
Just added FAKE targets to version (from RELEASE_NOTES) and publish (from `build` folder).
publish takes optional tag:
`./build Publish --tag=next`

**Comment by et1975** [comment]
Closing thanks to the switch to paket.
